### ISSUE TITLE
how add random food

### ISSUE THREAD

**Issue by mafiakb** [OgarPlus/Ogar-Plus#3]
how add random food in FFA?

**Comment by DaFudgeWizzad** [comment]
By random food you mean...?

**Comment by mafiakb** [comment]
Yes i do @DaFudgeWizzad i want play in FFA food x2 x5 x10 x1 random. Is possible?

**Comment by DaFudgeWizzad** [comment]
It is very possible :) I will get to work!

**Comment by mafiakb** [comment]
Oh ty :D

**Comment by DaFudgeWizzad** [comment]
I've been working on this for a while now. It's hard to find where the food mass is specified, but I got it. You will be able to turn it on and off in the config (So you can use it for any game mode) in 1 - 2 days, since I'm a bit busy.

**Comment by Porama6400** [comment]
I'll make gamemode for that
any good name?

FFARandom
......

**Comment by mafiakb** [comment]
Yeah :D

**Comment by Agarian** [comment]
In GameServer.js, find this line:

    GameServer.prototype.spawnFood = function() {

then edit the next line which goes

    var f = new Entity.Food(this.getNextNodeId(), null, this.getRandomPosition(),       this.config.foodMass);

into

    var f = new Entity.Food(this.getNextNodeId(), null, this.getRandomPosition(), this.config.foodMass         * Math.random() >> 0);

Math.random() function will generate a double from 0 to 1, and accompanied with a multiplier, ex. 50, will generate you a number from 0 to 50. >> 0 will round the number.
